sonic-utilities-tests/filter_fdb_input/test_vectors.py

```diff
@@ -61,9 +69,52 @@
             },
         ],
     },
+    {
+        "arp":[
+            {
+                "NEIGH_TABLE:Vlan1000:192.168.0.10": {
```

Comment:
do we need to add Vlan1 entry here to be filtered out?

Reply:
I will add more test cases to that effect